dll/win32/fmifs/query.c

```diff
@@ -57,27 +57,27 @@ QueryAvailableFileSystemFormat(
  * @param[in] DriveRoot
  * String which contains a DOS device name,
  *
- * @param[in,out] DeviceInformation
- * Pointer to buffer with DEVICE_INFORMATION structure which will receive data.
+ * @param[out] DeviceInformation
+ * Pointer to buffer which will receive DEVICE_INFORMATION data.
  *
  * @param[in] BufferSize
- * Size of buffer in bytes.
+ * Size of DeviceInformation buffer, in bytes.
  *
  * @return
- * TRUE if the buffer was large enough and was filled with
+ * TRUE if the buffer was large enough (pre-Vista at least, Vista+ if possible) and was filled with
```

Comment:
```suggestion
 * TRUE if the buffer was large enough at least for DeviceFlags and was filled with
```

Reply:
Yeah, though I think my proposal is more explicit about current implementation supporting two/both behaviors (especially the Vista+ one even on NT5.2 build),
`DeviceFlags` would be more explicit about the actual field (which callers should know anyway), but misses a reference to Vista threshold,
so I would rather keep it, unless you insist..

Reply:
It's just more clear IMO, and having the Vista behaviour documented in a remark and code should be sufficient. I am not insisting though.